governance/votes/Executive vote - November 26, 2021.md

```diff
@@ -0,0 +1,150 @@
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** and **additions** will occur within the Maker Protocol:
+- The **wBTC-C** vault type will be onboarded to the Maker Protocol with the parameters below.
+- The **GUSD peg stability module** will be onboarded to the Maker Protocol with the parameters below.
+- The **Surplus Buffer** will be increased $fill_in
+- The **Aave DAI direct deposit module** Debt Ceiling will be increased from 50 million DAI to 100 million DAI.
+- **906,805 DAI** will be transferred to [SAS-001](https://mips.makerdao.com/mips/details/MIP39c2SP25), [IS-001](https://mips.makerdao.com/mips/details/MIP39c2SP24) and [DECO-001](https://mips.makerdao.com/mips/details/MIP39c2SP23) as per their budget proposals.
+- **9,173,516 DAI** will be streamed to [RWF-001](https://mips.makerdao.com/mips/details/MIP39c2SP1), [COM-001](https://mips.makerdao.com/mips/details/MIP39c2SP8) [SAS-001](https://mips.makerdao.com/mips/details/MIP39c2SP25), [IS-001](https://mips.makerdao.com/mips/details/MIP39c2SP24), [MKT-001](https://mips.makerdao.com/mips/details/MIP39c2SP5) and [DECO-001](https://mips.makerdao.com/mips/details/MIP39c2SP23) as per their budget proposals.
+
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots in the event of early liquidations on new collateral types.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### wBTC-C Vault Type Onboarding
+
+As per the successful outcome of this [governance poll](https://vote.makerdao.com/polling/QmdVYMRo?network=mainnet#poll-detail), wBTC-C will be onboarded with the parameters listed below if this executive proposal passes.
+
+**Collateral Parameters**
+
+* Underlying Collateral: Wrapped Bitcoin (WBTC)
```

Comment:
capitalised WBTC here, needs to be consistent whichever way you choose to go with it.

Reply:
Need is a strong word, and technically the token and the vault type are different.

Reply:
Actually the ilk should be capitalized more than the collateral.

Reply:
Ahha! Now I've swapped both of them.

Reply:
Yeah fair, need was a bit strong. Probably hungry and cranky when I wrote that. Sorry.